scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/Conversion.scala

```diff
@@ -0,0 +1,143 @@
+package scala.meta
+package internal.semantic
+
+import java.nio.charset.Charset
+import java.nio.file.Files
+import scala.collection.immutable.Seq
+import scala.meta.inputs.{Input => mInput}
+import scala.meta.inputs.{Position => mPosition}
+import scala.meta.internal.io._
+import scala.meta.internal.semantic.vfs.Paths
+import scala.meta.internal.semantic.{schema => s}
+import scala.meta.internal.semantic.{vfs => v}
+import scala.meta.io._
+import scala.meta.{Dialect => mDialect}
+import scala.meta.{semantic => m}
+
+object SchemaToMetaConversion {
```

Comment:
Does this commit simply move the logic around, or it moves the logic + applies some changes? If it's the latter, I would find it useful to have this commit split into several commits.